core/selection.js

```diff
@@ -1168,7 +1168,7 @@
 					range = sel.createRange();
 
 				// The selection range is reported on host, but actually it should applies to the content doc.
-				if ( sel.type != 'None' && range.parentElement().ownerDocument == doc.$ )
+				if ( sel.type != 'None' && ( range.parentElement() || {}).ownerDocument == doc.$ )
```

Comment:
We could also check if `range.parentElement()` exists, not sure which one is more readable. Please remember about proper formatting like `( range.parentElement() || {} )` (missing space before closing parenthesis).